changelog.md

```diff
@@ -41,4 +41,6 @@
   - Upgrade to brod 3.7.11 to fix a bug when receiving empty batch from compacted transactions
 1.7.4
   - Remove expire_time from GUI since kafka 2.0 does not support it any more
+1.x.y
```

Comment:
This doesn't look right.

Reply:
I was assuming you are not going to tag this commit and release.
the next could be 1.8.0 or 1.7.5 when you want to release.

Reply:
I would have no problem tagging this as 1.7.5, but WRT to small changes not necessarily warranting a release, we should have a crystal clear convention for pending release notes.  My preference is to write them newest first and to omit the VSN until it's time to tag and release.  But this won't work here since the changelog is newest-last, and omitting a VSN would just look like a stray blank line, or some preliminary gibberish (which is how I interpreted your 1.x.y).

Reply:
I can revert the changlog to newest-frist.